Scripts/Python/xOptionsMenu.py

```diff
@@ -1834,23 +1765,20 @@ def LoadKeyMap(self):
         km = ptKeyMap()
         KeyMapString = self.getChronicleVar("KeyMap")
         if not KeyMapString:
-            PtDebugPrint("xOptionsMenu.LoadKeyMap():\tHmm... Empty chronicle...")
+            PtDebugPrint("xOptionsMenu.LoadKeyMap():\tHmm... Empty chronicle... Setting to default.")
+            self.ISetDefaultKeyMappings()
             return
 
-        KeyMapArray = KeyMapString.split()
         # set the key binds back to the saved
-        for counter, control_code in enumerate(defaultControlCodeBindsOrdered):
-            if isinstance(control_code, str):
-                key1 = KeyMapArray[counter]
-                PtDebugPrint("Binding " + key1 + " to " + control_code)
-                km.bindKeyToConsoleCommand(key1,control_code)
+        for controlCode, mappedKey in zip(kDefaultControlCodeBinds, KeyMapString.split(" ")):
+            if isinstance(controlCode, str):
+                PtDebugPrint(f"xOptionsMenu.LoadKeyMap(): Binding {mappedKey=} to {controlCode=}", level=kWarningLevel)
+                km.bindKeyToConsoleCommand(mappedKey, controlCode)
             else:
-                controlStr = km.convertControlCodeToString(control_code)
-                SubArray = KeyMapArray[counter].split("$")
-                key1 = SubArray[0]
-                key2 = SubArray[1]
-                PtDebugPrint("Binding " + key1 + " & " + key2 + " to " + controlStr)
-                km.bindKey(key1,key2,controlStr)
+                controlStr = km.convertControlCodeToString(controlCode)
+                keys = mappedKey.split("$")
+                PtDebugPrint(f"xOptionsMenu.LoadKeyMap(): Binding {keys=} to {controlStr=}", level=kWarningLevel)
+                km.bindKey(*keys, controlStr)
```

Comment:
To make this more robust in case the chronicle text isn't formatted correctly:

```suggestion
                key1, _, key2 = mappedKey.partition("$")
                PtDebugPrint(f"xOptionsMenu.LoadKeyMap(): Binding {key1=} & {key2=} to {controlStr=}", level=kWarningLevel)
                km.bindKey(key1, key2, controlStr)
```